pkgs/tools/networking/wg-netmanager/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, stdenv, fetchFromGitHub, rustPlatform, darwin, wireguard-go }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "wg-netmanager";
+  version = "0.3.8";
+
+  src = fetchFromGitHub {
+    owner = "gin66";
+    repo = "wg_netmanager";
+    rev = "wg_netmanager-v${version}";
+    sha256 = "1fbrgn1hnygfy337r52dhrq8h2lzdr0b675ys738ibiwwm6s558z";
+  };
+
+  # related to the Cargo.lock file
+  cargoSha256 = "0wj6rlj5iyzgx5xxzd4mmza2kiwfnpyi918w7gb03hqppzd2xkrr";
+
+  buildInputs = lib.optional stdenv.isDarwin darwin.apple_sdk.frameworks.Security;
```

Comment:
```suggestion
  buildInputs = lib.optional stdenv.isDarwin Security;
```

Reply:
changed accordingly

Reply:
Please take a look what other similar programs do - Security should be inherited in top-level.nix.

Reply:
I have copied it from another package like

https://github.com/NixOS/nixpkgs/blob/master/pkgs/tools/networking/boringtun/default.nix

Reply:
Yeah, thats wrong, too.

Reply:
If I use other rust example (like dogdns) I get this error. Here I have copied dogdns/default.nix and only change the package name. Perhaps the upstream nix-build system is broken ?:
```
error: anonymous function at /home/jochen/src/nixpkgs/pkgs/tools/networking/wg-netmanager/default.nix:1:1 called without required argument 'Security'

       at /home/jochen/src/nixpkgs/lib/customisation.nix:69:16:

           68|     let
           69|       result = f origArgs;
             |                ^
           70|

       … while evaluating 'makeOverridable'

       at /home/jochen/src/nixpkgs/lib/customisation.nix:67:24:

           66|   */
           67|   makeOverridable = f: origArgs:
             |                        ^
           68|     let

       … from call site

       at /home/jochen/src/nixpkgs/lib/customisation.nix:121:8:

          120|       auto = builtins.intersectAttrs (lib.functionArgs f) autoArgs;
          121|     in makeOverridable f (auto // args);
             |        ^
          122|

       … while evaluating 'callPackageWith'

       at /home/jochen/src/nixpkgs/lib/customisation.nix:117:35:

          116|   */
          117|   callPackageWith = autoArgs: fn: args:
             |                                   ^
          118|     let

       … from call site

       at /home/jochen/src/nixpkgs/pkgs/top-level/all-packages.nix:10759:19:

        10758|
        10759|   wg-netmanager = callPackage ../tools/networking/wg-netmanager { };
             |                   ^
        10760|

```